XRTK.SDK/Packages/com.xrtk.sdk/Features/Input/Handlers/ManipulationHandler.cs

```diff
@@ -627,6 +628,16 @@ protected virtual float CalculateNudgeDistance(InputEventData<Vector2> eventData
             return prevExtent + nudgeAmount * (eventData.InputData.y < 0f ? -1 : 1);
         }
 
+        protected virtual Vector3 CalculateScaleAmount(InputEventData<Vector2> eventData, Vector3 prevScale)
```

Comment:
Wondering if we should do the same for the rotation calculation as well, thoughts?

Reply:
That would be a good idea, yes

Reply:
We can add this later if needed.